src/netspresso_trainer/dataloaders/augmentation/custom.py

```diff
@@ -351,6 +354,130 @@ def __repr__(self):
         return format_string
 
 
+class RandomErasing(T.RandomErasing):
+
```

Comment:
class variable `visualize`

If not yet implemented for visualize, please set this as False .